src/sidebar/components/test/group-list-item-test.js

```diff
@@ -29,6 +41,10 @@ describe('GroupListItem', () => {
       orgName: sinon.stub(),
     };
 
+    fakeGroups = {
```

Comment:
For me this highlighted the confusion in naming overload for `groups`: it can mean either the groups service or a list of groups. In this case, I'm assuming it's the service.

Reply:
I'll rename it to `fakeGroupsService` where possible.